test/test_install/CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ if(GINKGO_BUILD_HIP)
         set (GINKGO_PIC_OPTION "-fPIC")
         set (GINKGO_CUDA_PIC_OPTION "-Xcompiler '-fPIC'")
     endif()
+    if (NOT GINKGO_BUILD_SHARED_LIBS AND CMAKE_BUILD_TYPE MATCHES [Dd][Ee][Bb][Uu][Gg])
+        set (GINKGO_PIC_OPTION "-fPIC")
+    endif()
```

Comment:
This does not play nice with multi-config generators. Using generator expressions would be the correct thing here, but that probably doesn't work with HIP (I tried configuring HIP with `-G"Ninja Multi-Config"` before, and that didn't work)

Reply:
I update it with $<$<CONFIG:Debug>:-fPIC>